README.md

```diff
@@ -34,11 +34,12 @@ To build the zeeguu-api-core container image:
 docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
 ```
 
-To build the zeeguu-web container image:
+To build the zeeguu-web container image:   
+(Make sure to replace 1.1.1.1:9001 with the url:port where your API can be reached from other clients.)
 ```sh
 docker build -t zeeguu-web --build-arg ZEEGUU_API__EXTERNAL="http://1.1.1.1:9001"  -f docker-files/zeeguu-web/Dockerfile .
 ```
-Make sure to replace ``1.1.1.1:9001`` with the url:port where your API can be reached from other clients.
+<!-- Make sure to replace ``1.1.1.1:9001`` with the url:port where your API can be reached from other clients. -->
```

Comment:
This should be removed instead of commented out.

Reply:
Ups, the comment was a mistake. made the changes